server/endpoint/endpoint.go

```diff
@@ -2,6 +2,7 @@ package endpoint
 
 import (
 	"github.com/giantswarm/microendpoint/endpoint/healthz"
+	"github.com/giantswarm/microendpoint/endpoint/version"
```

Comment:
While enabling the v2 resource set I noticed that the version endpoint was not set up.

Even though chart-operator currently runs in guest clusters I think it should have the endpoint. Also later when we run it in the host cluster it will be needed.